pyproject.toml

```diff
@@ -50,7 +50,7 @@ embeddings = [
     "tiktoken~=0.7.0"
 ]
 agentops = ["agentops>=0.3.0"]
-fastembed = ["fastembed>=0.4.1"]
+fastembed = ["fastembed>=0.7.0"]
```

Comment:
i feel like we can drop this? no files use this, outside of `fastembed.py` but nothing is using it iirc

Reply:
That's awesome! I'm going to double check that